### ISSUE TITLE
Transform Mesh

### ISSUE THREAD

**Issue by yanxian-ll** [CloudCompare/CloudComPy#196]
Hello, are there any methods to transform mesh here? 
Like，‘applyRigidTransformation’ function.

**Comment by prascle** [comment]
Hello, 
When you want to modify a mesh with `scale` or `applyRigidTransformation`, you have to get the mesh associated point cloud (method `getAssociatedCloud`). When you use the `scale` or `applyRigidTransformation` method on the associated cloud, the mesh is also transformed.
Does this answer your question?
Best regards,
Paul

**Comment by yanxian-ll** [comment]
Deeply appreciate it!

**Comment by stellarpower** [comment]
Would it be appropriate to add a quick mention of that to the document in /sphinxDoc/userEntities.rst at all? E.g. at the top of the mesh section, or perhaps a note next to the translation, rotation, etc. section for clouds noting the parallel for meshes. I was also searching for a few minutes until I found the solution. 

Also, I don't know if it would have any consequences - would adding a binding to the mesh class that performed the same operation on the associated cloud be possible? I've just opened up a file so was hoping there'd be an obvious method to call, didn;t spot this looking through the output of `dir`.

Thanks
